control_panel_api/views.py

```diff
@@ -25,7 +25,9 @@
     GroupSerializer,
     S3BucketSerializer,
     UserSerializer,
-    UserS3BucketSerializer)
+    UserS3BucketSerializer,
+    AppUserSerializer
```

Comment:
As per other imports (and lists where order is not important), I'd keep this alphabetically sorted and with a trailing comma.